src/nett/nett.py

```diff
@@ -101,13 +101,13 @@ def run(self,
         self.logger.info(f"Set up run directory at: {output_dir.resolve()}")
 
         # calculate iterations
-        iterations: dict[str, int] = {
-            "train": steps_per_episode * train_eps,
-            "test": test_eps * self.environment.num_test_conditions
-        }
-
-        if not issubclass(self.brain.algorithm, RecurrentPPO):
-            iterations["test"] *= steps_per_episode
+        iterations: dict[str, int] = {}
+        if mode in ["train", "full"]:
```

Comment:
```suggestion
        if mode in ["train", "full"] or job_memory == "auto": #TODO: auto job_memory runs a train job, requiring iterations["train"] to run
```
auto memory estimation currently can only use a "train" job to estimate memory usage (which will always use more memory than a "test" job so it is the safer of the two options). Until auto memory estimation is adjusted to support accurate test job memory usage, I recommend adding this.